larpandoracontent/LArControlFlow/NeutrinoIdTool.h

```diff
@@ -259,6 +267,9 @@ class NeutrinoIdTool : public SliceIdBaseTool
     float m_minProbability;      ///< Minimum probability required to classify a slice as the neutrino
     unsigned int m_maxNeutrinos; ///< The maximum number of neutrinos to select in any one event
 
+    // Persistence
```

Comment:
Unnecessary, I'd say, already covered by comment attached to member variable below.